src/bootloader/simple_pg.md

```diff
@@ -387,6 +402,19 @@ pub const PageAttribute = enum {
     executable,
 };
 
+pub const PageError = error{
```

Comment:
It's preferable that the definitions of errors are not listed here to reduce the size of code snippet.
Readers are expected to reference the repository for these definitions.

Reply:
I shrunk the definition to a single line. ... Maybe it's because I'm also completely new to Zig, but I feel it would be nice if looking into the repo wouldn't be necessary to get working code. Also it takes me out of the flow a bit every time. ... But if you think this should still be removed, tell me.

Reply:
> but I feel it would be nice if looking into the repo wouldn't be necessary to get working code.

Understood. It makes some sense :+1:  You can write the definition not in a single line (or you have to remove the trailing comma).